### DISCUSSION TITLE
AutoKit - Bluetooth Audio interruptions

### DISCUSSION THREAD

**Discussion by mbhforum** [tesla-android/issue-tracker#31] [Q&A]
Running On a 2019 Tesla Model S and one issue I am having is when I receive a iMessage while listening to SiriusXM/Spotify, the music stops and plays my iMessage but when I am done listening, it does not play my music automatically and I have to press play in the CarPlay app (eg: Spotify).  Is this a known issue or CarPlay issue?

**Comment by mikegapinski** [comment]
That might be a collision between Tesla BT integration and CarPlay. 

I would disable the announce messages via Siri and use Tesla message integration. (that's the setup I use, Messages via Siri are more limited then Tesla)

**Reply by mikegapinski** [reply]
Audio with CarPlay(navigation sounds, switching etc) will work better in next builds, I am working on it now

**Reply by mbhforum** [reply]
Awesome! Great
Job so far! One of my biggest reasons for using CarPlay is group text messaging support which Tesla doesn't support.

**Reply by mbhforum** [reply]
Just an FYI, this is still an issue on the latest build.

**Reply by mikegapinski** [reply]
Feel free to contact Carlinkit, it's an issue with their dongle, I am not directly integrating with it. Do you have the music autoplay settings turned on in Autokit? That might change something in this regard. 

Given that they've fixed a lot of issues with the dongle I hope that what you are experiencing will get resolved one day
